ignite/cmd/relayer_configure.go

```diff
@@ -477,16 +486,15 @@ func initChain(
 		return nil, errors.Wrapf(err, "cannot resolve %s", name)
 	}
 
-	session.StopSpinner()
-
 	accountAddr := account.Address(addressPrefix)
 
+	session.StopSpinner()
 	session.Printf("🔐  Account on %q is %s(%s)\n \n", name, accountName, accountAddr)
 	session.StartSpinner(color.Yellow.Sprintf("trying to receive tokens from a faucet..."))
 
 	coins, err := c.TryRetrieve(cmd.Context())
-	session.StopSpinner()
 
+	session.StopSpinner()
 	session.Print(" |· ")
```

Comment:
What's the purpose of this characters ` |· ` ?

Reply:
Output formatting
Ex:
```
ignite  n  reward release 1 "http://0.0.0.0:26657" --spn-node-address http://0.0.0.0:26661 --spn-faucet-address http://0.0.0.0:4502 --testnet-prefix spn

𝓲 Network client: 07-tendermint-0
𝓲 Testnet chain spn-2 client: 07-tendermint-0

🔐  Account on "source" is default(spn165wt8n7n9280vqrgkjz85d4ew90fe3vt6cg8v6)

received coins from a faucet
 |· (balance: 199667uspn)

```